curriculum/challenges/english/25-front-end-development/quiz-relational-database/66f1af82732957c895f0b21a.md

```diff
@@ -17,439 +17,440 @@ Answer all of the questions below correctly to pass the quiz.
+List of all employee IDs
 
 ---
 
-Placeholder distractor 3
+Average salary of all employees
 
 #### --answer--
 
-Placeholder answer
+List of all employee names
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+How do you delete a column from a table in SQL?
 
 #### --distractors--
 
-Placeholder distractor 1
+REMOVE COLUMN
 
 ---
 
-Placeholder distractor 2
+DELETE COLUMN
 
 ---
 
-Placeholder distractor 3
+ALTER TABLE
 
 #### --answer--
 
-Placeholder answer
+DROP COLUMN
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is the function of the PRIMARY KEY constraint in SQL?
 
 #### --distractors--
 
-Placeholder distractor 1
+To link two tables together
 
 ---
 
-Placeholder distractor 2
+To define the default value for a column
 
 ---
 
-Placeholder distractor 3
+To allow null values in the column
 
 #### --answer--
 
-Placeholder answer
+To ensure that the column contains unique values
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What does the PostgreSQL command \c do?
 
 #### --distractors--
 
-Placeholder distractor 1
+Lists all tables in the current database
 
 ---
 
-Placeholder distractor 2
+Displays the current database connection details
 
 ---
 
-Placeholder distractor 3
+Exits the psql shell
 
 #### --answer--
 
-Placeholder answer
+Connects to the specified database
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What type of relationship does this foreign key create between two tables?
 
 #### --distractors--
 
-Placeholder distractor 1
+One-to-One
 
 ---
 
-Placeholder distractor 2
+Many-to-One
 
 ---
 
-Placeholder distractor 3
+Many-to-Many
 
 #### --answer--
 
-Placeholder answer
+One-to-Many
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which command inserts data into a table?
 
 #### --distractors--
 
-Placeholder distractor 1
+PUT VALUES
 
 ---
 
-Placeholder distractor 2
+UPDATE INTO
 
 ---
 
-Placeholder distractor 3
+ADD DATA
 
 #### --answer--
 
-Placeholder answer
+INSERT INTO
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+How do you rename a column in SQL?
 
 #### --distractors--
 
-Placeholder distractor 1
+MODIFY COLUMN
 
 ---
 
-Placeholder distractor 2
+CHANGE COLUMN
 
 ---
 
-Placeholder distractor 3
+ALTER COLUMN
 
 #### --answer--
 
-Placeholder answer
+RENAME COLUMN
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which query will delete all records from the employees table?
 
 #### --distractors--
 
-Placeholder distractor 1
+REMOVE ALL
 
 ---
 
-Placeholder distractor 2
+TRUNCATE employees
 
 ---
 
-Placeholder distractor 3
+DROP RECORDS
 
 #### --answer--
 
-Placeholder answer
+DELETE FROM employees
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which SQL constraint ensures that a column cannot contain duplicate values?
 
 #### --distractors--
 
-Placeholder distractor 1
+NOT NULL
 
 ---
 
-Placeholder distractor 2
+FOREIGN KEY
 
 ---
 
-Placeholder distractor 3
+PRIMARY KEY
```

Comment:
There's previously question: _What is the function of the PRIMARY KEY constraint in SQL?_, with answer _To ensure that the column contains unique values_. That makes look, like `PRIMARY KEY` also might be considered as an answer here.